### PR TITLE
Add gosuslugi.ru

### LINKED ISSUE #6159: Add Госуслуги (Gosuslugi)
### Site name

Портал государственных и муниципальных услуг Российской Федерации (Госуслуги, Gosuslugi)

### Site URL

https://www.gosuslugi.ru/

### Documentation URL

_No response_

### Supported 2FA methods

- [ ] TOTP (RFC-6238, Google Authenticator)
- [ ] Proprietary Software (Authy, Duo, etc.)
- [ ] FIDO2/U2F/WebAuthn security keys
- [X] Proprietary hardware (Yubikey OTP, RSA keys, etc.)
- [ ] Phone calls
- [X] SMS tokens
- [ ] Email tokens

### Eligibility

- [X] I have checked that the site meets 2fa.directory's [contributing guidelines and site criteria](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md)
- [X] The site is ranked within the top 200,000 sites on [Alexa](https://www.alexa.com/siteinfo/) or has a substantial social media following.
- [X] The site does not contain or promote content that violates 2fa.directory's [excluded categories and websites guidelines](https://github.com/2factorauth/twofactorauth/blob/master/EXCLUSION.md), including pornographic, discriminatory, or unlawful content.
- [X] The issue I'm creating is not a duplicate of an existing [issue](https://github.com/2factorauth/twofactorauth/issues).
- [X] The issue I'm creating is not a duplicate of an existing [pull request](https://github.com/2factorauth/twofactorauth/pulls)

### Additional information

<details>
<summary>Screenshot</summary>

image

</details>

### PR DESCRIPTION
**Pull request by Carlgo11**
Closes #6159

### CODE REVIEW

**Comment by rakleed** [comment]
@Carlgo11 why haven't you merged this and #6161 pull requests yet? Due to the fact that there is no region "Russia"?

But it's not difficult to add it. Or should I do it?

**Comment by Carlgo11** [comment]
@rakleed I can't merge my own PR. It needs to be approved and merged by two other maintainers.

As for why my PR hasn't been reviewed yet I don't know 🤷

**Comment by rakleed** [comment]
@ApCoder123 @phallobst please take a look at this and #6161 PRs.

**Review by kmpoppe — APPROVED** [review]

File: img/g/gosuslugi.ru.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...-2.1l.2-.3V1.2c0-.2-.1-.4-.3-.4M95.2.9h-6a132 132 0 0 1-5.7 16.6L77.5 1l-.3-.2h-6l-.3.1v.3l9 24.6-2.6 5-1.8 3.4v.3l.3.2h6.5l.3-.2a247.5 247.5 0 0 0 13-33.2l-.1-.3-.3-.1m21.4 18.9-.2-.2h-.3c-1.4.5-4.1 1-6 1-4.1 0-6-1.1-6-7.4 0-5 .6-7.4 6-7.4 1.5 0 3 .2 4.8.8.2 0 .3 0 .4-.2l2.4-4.8v-.3l-.2-.2a26 26 0 0 0-7.8-1.2c-8.6 0-12.4 4-12.4 13.1 0 9.2 3.8 13.3 12.4 13.3a30 30 0 0 0 8.4-1.3c.2-.1.2-.2.2-.4l-1.7-4.8zM172.5.9h-6l-.3.2a132 132 0 0 1-5.4 16.4l-6-16.5-.3-.2h-6l-.3.1v.3l9 24.6-2.6 5-1.8 3.4v.3l.3.2h6.5l.3-.2a223.6 223.6 0 0 0 12.9-33.2l-.1-.3-.2-.1m22-.1h-17.4c-.2 0-.3.1-.3.3v24.1c0 .2.1.3.3.3h5.9c.2 0 .3-.1.3-.3V6.6h9.1l.3-.2 2.1-5.1V1l-.3-.2"/><path fill="#0065b1" d="M31.3 20.8c-3.9 0-5.1-1.1-5.1-7.6 0-7 1.3-7.6 5.1-7.6s5.2.6 5.2 7.6c0 6.6-1.3 7.6-5.2 7.6m0-20.7c-8.5 0-11.8 3.6-11.8 13 0 9.5 3.3 13.2 11.8 13.2s11.9-3.7 11.9-13.2c0-9.3-3.4-13-11.9-13m35.3 19.7-.2-.2h-.3c-1.4.5-4.1 1-6 1-4.1 0-6-1.1-6-7.4 0-5 .6-7.4 6-7.4 1.5 0 3 .2 4.8.8.1 0 .3 0 .4-.2l2.4-4.8v-.3l-.2-.2A28 28 0 0 0 59.8 0c-8.6 0-12.4 4-12.4 13.1 0 9.2 3.8 13.3 12.4 13.3a30 30 0 0 0 8.4-1.3c.2-.1.2-.2.2-.4l-1.8-4.9zM17.7.8H.3C.1.8 0 1 0 1.2v24.1c0 .2.1.3.3.3h5.9c.2 0 .3-.1.3-.3V6.6h9.1l.3-.2L18 1.3V1l-.3-.2"/></svg>
```
Comment (phallobst) [inline img/g/gosuslugi.ru.svg]: The logo for the mobile app might look better in our default square(-ish) format:

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 194 213" xmlns:v="https://vecta.io/nano"><path fill="#ef3e58" d="M162.8 108.7h-6.4c-.2 0-.4.2-.4.4v20.3c-1.4.4-2.8.6-4.4.6-4.2 0-5.2-1.3-5.2-6.9v-14c0-.2-.2-.4-.4-.4h-6.4c-.2 0-.4.2-.4.4v14.7c0 9.2 3.1 12.7 11 12.7 4.5 0 9.6-1.2 12.6-2.3.1-.1.2-.2.2-.3v-24.8c.2-.2 0-.4-.2-.4zm-135.6.1h-6.5c-.2 0-.3.1-.3.3-1 4-3.1 10.8-5.9 17.9L8 109c-.1-.1-.2-.2-.3-.2H1.1c-.1 0-.2.1-.3.2s-.1.2 0 .3l9.8 26.8c-1 2.1-1.9 3.8-2.8 5.4-.7 1.2-1.3 2.4-1.9 3.7-.1.1-.1.2 0 .3s.2.2.3.2h7.1c.1 0 .3-.1.3-.2 1.2-2.3 2.7-5.5 4.1-8.8 4.2-9.8 7.5-19 10-27.4 0-.1 0-.2-.1-.3-.2-.2-.3-.2-.4-.2zm23.3 20.6c0-.1-.1-.2-.2-.2H50c-1.6.6-4.5 1-6.5 1-4.5 0-6.6-1.2-6.6-8 0-5.5.7-8 6.6-8 1.7 0 3.2.2 5.2.8.2 0 .3 0 .4-.2l2.6-5.2v-.3c0-.1-.1-.2-.2-.2-2.6-.8-5.7-1.3-8.5-1.3-9.3 0-13.5 4.4-13.5 14.3 0 10 4.2 14.5 13.5 14.5 2.3 0 7-.5 9.2-1.4.2-.1.3-.3.2-.5l-1.9-5.3zm60.9-20.6h-6.5c-.2 0-.3.1-.3.3-1 4.1-3.1 10.8-5.9 17.9l-6.5-18c-.1-.1-.2-.2-.3-.2h-6.6c-.1 0-.2.1-.3.2s-.1.2 0 .3l9.8 26.8c-1 2.1-1.9 3.8-2.8 5.4-.7 1.2-1.3 2.4-1.9 3.7-.1.1-.1.2 0 .3s.2.2.3.2h7.1c.1 0 .3-.1.3-.2 1.2-2.3 2.7-5.5 4.1-8.8 4.1-9.8 7.5-19 10-27.4 0-.1 0-.2-.1-.3-.2-.2-.3-.2-.4-.2zm24-.1h-18.9c-.2 0-.4.2-.4.4v26.3c0 .2.2.4.4.4h6.4c.2 0 .4-.2.4-.4V115h9.9c.1 0 .3-.1.3-.2.8-1.8 1.5-3.7 2.3-5.6v-.3c-.2-.1-.3-.2-.4-.2z"/><path fill="#0f67b1" d="M34.8 68.3c-9.2 0-12.8 4-12.8 14.1 0 10.3 3.6 14.3 12.8 14.3 9.3 0 12.9-4 12.9-14.3 0-10.1-3.6-14.1-12.9-14.1zm0 22.6c-4.2 0-5.5-1.1-5.5-8.3 0-7.6 1.4-8.3 5.5-8.3 4.2 0 5.6.7 5.6 8.3 0 7.1-1.3 8.3-5.6 8.3zm38.5-1.1c0-.1-.1-.2-.2-.2h-.3c-1.6.6-4.5 1-6.5 1-4.5 0-6.6-1.2-6.6-8 0-5.5.6-8 6.6-8 1.7 0 3.2.2 5.2.8.2 0 .3 0 .4-.2l2.6-5.2v-.3c0-.1-.1-.2-.2-.2-2.6-.8-5.7-1.3-8.5-1.3-9.3 0-13.5 4.4-13.5 14.3 0 10 4.2 14.5 13.5 14.5 2.3 0 7-.5 9.2-1.4.2-.1.3-.3.2-.5l-1.9-5.3zM1.1 69.1c-.2 0-.4.2-.4.4v26.3c0 .2.2.4.4.4h6.4c.2 0 .4-.2.4-.4V75.4h9.9c.1 0 .3-.1.3-.2.8-1.8 1.5-3.7 2.3-5.6v-.3c-.1-.1-.2-.2-.3-.2h-19z"/><linearGradient id="A" x1="100.52" x2="100.52" y1=".88" y2="212.96" gradientUnits="userSpaceOnUse"><stop offset=".4" stop-color="#1466ac"/><stop offset=".66" stop-color="#ef4058"/></linearGradient><path fill="url(#A)" fill-rule="evenodd" d="M192.7 133v-.4c0-.2 0-.5.1-.7.3-3.7.5-8.2.7-13.4v-.4l.2-11.2c0-3.9-.1-7.7-.2-11.2v-.4l-.7-13.4c0-.2 0-.5-.1-.7v-.4c0-.4-.1-.7-.1-1.1v-.4c-.3-3.9-.5-6.3-.6-6.5 0-.4-.1-.8-.1-1.1v-.3c-.1-.7-.2-1.3-.4-2 0-.1 0-.2-.1-.3-1.7-8.1-6.2-17-11.8-23.7-2.2-2.6-4.5-4.9-6.9-6.7a318.4 318.4 0 0 0-28.1-18.5C129.9 11.5 116.7 5.5 115 4.7c-.1 0-.1-.1-.1-.1-3.9-1.7-8.6-2.9-13.7-3.4-1-.1-2-.2-3-.2-.9 0-1.8-.1-2.7-.1h-1.4c-6.9.1-13.2 1.4-18.2 3.7-.8.3-14.1 6.4-29.2 15.2l-.6.3A364.2 364.2 0 0 0 18 38.7c-3.8 2.8-7.2 6.4-10.3 11-.7 1.1-.7 3.7 2.3 3.7h7.7c3.3 0 4.1-2 7.8-4.7 4.1-3 12.2-9 27-17.8C63.2 24.6 73.2 19.7 78 17.4c.1 0 .2-.1.3-.1 0 0 .1 0 .1-.1 0 0 .1 0 .1-.1.1 0 .2-.1.2-.1.1 0 .2-.1.3-.1 1.2-.6 2-.9 2-.9 3.4-1.5 8.2-2.5 13.3-2.6h2c1.5 0 3 .1 4.4.3 3.1.4 5.9 1 8.2 2 .2.1.5.2.7.3a338.13 338.13 0 0 1 28.6 15 322.77 322.77 0 0 1 27 17.8c1.9 1.4 3.8 3.4 5.6 5.6 4.5 5.8 8.2 13.7 8.8 19.8 0 .1.3 2.5.6 6.6 0 .3 0 .5.1.8v.6c0 .4.1.7.1 1.1v.4c0 .2 0 .5.1.7 0 .5.1 1 .1 1.5 0 .3 0 .6.1.9v.2c0 .4 0 .7.1 1.1v.8a317.08 317.08 0 0 1 0 35.8v.8c0 .4 0 .7-.1 1.1v.2c0 .3 0 .6-.1.9 0 .5-.1 1-.1 1.5 0 .3 0 .5-.1.7v.4c0 .4-.1.7-.1 1.1v.6c0 .3 0 .6-.1.8-.3 4.1-.6 6.6-.6 6.6-.6 6.1-4.3 14-8.8 19.8-1.8 2.3-3.7 4.2-5.6 5.6-.1 0-2.7 2-7 5a322.42 322.42 0 0 1-48.2 27.6l-.4.2c-.2.1-.5.2-.7.3-2.3.9-5.1 1.6-8.2 2-1.4.2-2.9.3-4.4.3h-2c-5.1-.1-9.9-1-13.3-2.6 0 0-.7-.3-2-.9-.1-.1-.3-.1-.4-.2s-.2-.1-.3-.2c-.1 0-.1-.1-.2-.1s-.2-.1-.3-.1c-.1-.1-.2-.1-.4-.2h-.1c-5.1-2.4-14.7-7.2-24.9-13.2-14.9-8.7-22.9-14.8-27-17.8-3.7-2.7-4.5-4.7-7.8-4.7H10c-3 0-3 2.6-2.3 3.6 3.1 4.6 6.5 8.2 10.3 11 .5.4 12.7 9.4 28.1 18.5.8.5 1.6 1 2.5 1.4 14.3 8.2 26.6 13.8 27.3 14.2 4.9 2.2 11.3 3.5 18.1 3.7h1.4c.9 0 1.8 0 2.7-.1 1 0 2-.1 3-.2 5.1-.5 9.9-1.7 13.7-3.4 0 0 .1 0 .1-.1 1.7-.8 14.9-6.9 29.6-15.5 8.3-4.9 15.6-9.7 20.7-13.2l7.4-5.3c2.4-1.8 4.7-4.1 6.9-6.7 5.6-6.7 10.1-15.6 11.8-23.7 0-.1 0-.2.1-.3.1-.7.3-1.4.4-2v-.3c.1-.4.1-.8.1-1.1 0-.2.3-2.6.6-6.5v-.4c.2-.1.2-.5.2-.8"/><path fill="#ef3e58" d="M79.9 108.7H60.3c-.2 0-.3.1-.4.3-.5 8.9-2.2 18.4-4.6 26.2 0 .1 0 .2.1.3s.2.1.3.1h6.7c.2 0 .3-.1.3-.2 1.9-6 3.4-13.8 4-20.4h6.4v20.3c0 .2.2.4.4.4h6.4c.2 0 .4-.2.4-.4V109c0-.1-.2-.3-.4-.3z"/></svg>
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Review by phallobst — APPROVED** [review]

**Pull request merged**